src/pages/zzang/CoffeeCard/CoffeeCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function CoffeeCard(props){
+    return(
+        <>
+        <li>
+            <div className="item">
+                    <a href="#">
+                        <img src={props.item.img} alt={props.item.title} width="250px" height="300px"/>
+                    </a>
+                </div>
+            <dd>{props.item.title}</dd>
+        </li>
+        </>
+    )
+}
+
```

Comment:
```suggestion

function CoffeeCard(props){
  return(
    <>
      <li>
        <div className="item">
          <a href="#">
            <img src={props.item.img} alt={props.item.title} width="250px" height="300px"/>
          </a>
        </div>
        <dd>{props.item.title}</dd>
      </li>
    </>
  )
}

```

제가 알기로 들여쓰기 convention이 스페이스 2칸으로 알고 있어서 코멘트 남겨 봅니다.